src/containers/CallIn/CallIn.js

```diff
@@ -172,118 +174,138 @@ class CallIn extends Component {
         `${this.state.repFirstName} ${this.state.repLastName} (${displayName(this.state)})` :
         `${this.state.repFirstName} ${this.state.repLastName} (${displayName(this.state)})`;
 
-        const callIn = (this.state.repLastName) ? (
-            <>
+        if (!this.state.repLastName) {
+            return (
+                <div className={styles.Loading}>
+                    <Spin />
+                </div>
+            )
+        }
+
+        if (this.state.status === 'covid_paused') {
+            return (
                 <div className={styles.CallIn}>
-                    <section>
+                    <Alert
+                        message="Please don't call this Member of Congress"
+                        description={`
+                            We are asking you to not call your Member of Congress so that 
+                            they can focus on the COVID-19 Outbreak.
+                        `}
```

Comment:
```suggestion
                        description={`
                            We are asking you to not call your Member of Congress at this time 
                            so that they can focus on the COVID-19 outbreak. Your monthly
                            call-in notifications will resume when the crisis abates.
                        `}
```